package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "dnsimple-support",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "jquery": "^3.4.1",
+    "lunr": "^2.3.6"
+  },
+  "devDependencies": {
+    "concurrently": "^4.1.0",
+    "@fortawesome/fontawesome-free": "^5.9.0",
+    "copy-webpack-plugin": "^5.0.3",
+    "webpack": "^4.35.0",
+    "webpack-cli": "^3.3.5"
+  },
+  "scripts": {
+    "live": "concurrently 'bundle exec nanoc live' 'rake compile'",
```

```suggestion
    "live": "concurrently 'bundle exec nanoc live' 'bundle exec rake compile'",
```

Otherwise I get this error

```
➜  dnsimple-support git:(dependencies-via-yarn) ✗ yarn live
yarn run v1.16.0
$ concurrently 'bundle exec nanoc live' 'rake compile'
[1] rake aborted!
[1] Gem::LoadError: You have already activated rake 12.3.2, but your Gemfile requires rake 12.3.1. Prepending `bundle exec` to your command may solve this.
[1] /Users/weppos/Code/dnsimple-support/Rakefile:2:in `<top (required)>'
[1]
[1] Caused by:
[1] LoadError: cannot load such file -- bundler/setup
[1] /Users/weppos/Code/dnsimple-support/Rakefile:2:in `<top (required)>'
[1] (See full trace by running task with --trace)
[1] rake compile exited with code 1
```

---

Yep, sorry. That's correct.